WolvenKit.Modkit/RED4/Tools/MlmaskTools.cs

```diff
@@ -362,7 +368,11 @@ private static void DecodeSingle(ref byte[] maskData, uint maskWidth, uint maskH
             var tileIndex = (widthInTiles * yTile) + xTile + tilesOffset;
 
             var paramOffset = tilesData[tileIndex * 2];
-            var paramBits = tilesData[(tileIndex * 2) + 1];
+
+            // IndexOutOfRange Prevention
+            var paramBits = tilesData.Length > (tileIndex * 2) + 1
```

Comment:
@seberoth Getting length less than expected on the array here.

Relates to #939